sphinx_gallery/scrapers.py

```diff
@@ -129,6 +157,29 @@ def matplotlib_scraper(block, block_vars, gallery_conf, **kwargs):
     return rst
 
 
+def _anim_rst(anim, image_path, gallery_conf):
+    from matplotlib.animation import ImageMagickWriter
+    # output the thumbnail as the image, as it will just be copied
+    # if it's the file thumbnail
+    fig = anim._fig
+    image_path = image_path.replace('.png', '.gif')
+    fig_size = fig.get_size_inches()
+    thumb_size = gallery_conf['thumbnail_size']
+    use_dpi = round(
+        min(t_s / f_s for t_s, f_s in zip(thumb_size, fig_size)))
+    # FFmpeg is buggy for GIFs
```

Comment:
Might want to see matplotlib/matplotlib#18093

Reply:
@dopplershift would you be up for making a quick PR to triage based on the `LooseVersion` of matplotlib which backend to use?

Reply:
Sure. #733